host/3.0/publish-appservice-stage.yml

```diff
@@ -39,11 +45,11 @@ steps:
       set -e
       docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice
 
-      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice $TARGET_REGISTRY/dotnet:$(TargetVersion)-appservice-stage$(StageNumber)
```

Comment:
I think doing it here means, we need to run this pipeline 7 times for each cloud. right?

Will it be simpler if we push to other clouds in a separate publish-otherclouds.yml that only takes privateversion, targetversion and cloudname as inputs.
basically this file pushes to public cloud 1 stage at a time (same as now)
we run publish.yml to push to stageless tags in public cloud. (same as now)
now we run publish-otherclouds.yml once for each cloud and each run will push to all 7 stages simultaneously.
will that be easier from release instructions perspective?

(the assumption is we are okay to deploy to all stages in other clouds, since we have already validated the image in public cloud)

Reply:
As its not a manual process keeping per stage is OK!

@divyagandhisethi - can we optimize to publish multiple stages in a single task in release pipeline?

Reply:
@balag0 - Created a separate yaml for sovereign clouds.